racksdb/web/ui/src/components/ComboBox.vue

```diff
@@ -0,0 +1,110 @@
+<!--Copyright (c) 2022-2023 Rackslab
+
+This file is part of RacksDB.
+
+SPDX-License-Identifier: GPL-3.0-or-later -->
+
+<script setup lang="ts">
+import { ref, computed, watchEffect } from 'vue'
+import type { Ref } from 'vue'
+import { useRouter } from 'vue-router'
+import type { Datacenter, Infrastructure } from '@/composables/RacksDBAPI'
+import {
+  Combobox,
+  ComboboxButton,
+  ComboboxInput,
+  ComboboxOption,
+  ComboboxOptions
+} from '@headlessui/vue'
+import { ChevronUpDownIcon, CheckIcon } from '@heroicons/vue/24/outline'
+
+const selectedItem: Ref<string | undefined> = ref()
+const route = useRouter()
```

Comment:
`useRouter()` returns a router, not a route. The variable should be renamed router to better match its content.

Reply:
I've made the change of the variable.